specification/oracle/Oracle.Database.Management/models/autonomousdatabase/autonomousDatabase.tsp

```diff
@@ -578,6 +578,22 @@ model ScheduledOperationsType {
   scheduledStopTime?: string;
 }
 
+@doc("The list of scheduled operations.")
+model ScheduledOperationsTypeUpdate {
+  @doc("Day of week")
+  dayOfWeek: DayOfWeekUpdate;
```

Comment:
This property is optional in Swagger
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/oracle/resource-manager/Oracle.Database/stable/2025-03-01/openapi.json#L12013